apps/www/package.json

```diff
@@ -16,22 +16,23 @@
     "@heroicons/react": "^1.0.6",
     "@mdx-js/loader": "^1.6.22",
     "@mdx-js/react": "^1.6.22",
+    "@next/bundle-analyzer": "^13.0.0",
     "@next/mdx": "^10.0.7",
     "@supabase/auth-ui-react": "^0.1.6",
     "@supabase/supabase-js": "^2.0.0",
     "classnames": "^2.3.1",
-    "config": "*",
     "common": "*",
+    "config": "*",
     "date-fns": "2.10.0",
     "dayjs": "^1.11.4",
     "globby": "^12.0.2",
     "gray-matter": "^4.0.2",
     "markdown-toc": "^1.2.0",
     "marked": "^4.0.12",
     "next": "latest",
+    "next-compose-plugins": "^2.2.1",
```

Comment:
@hkaur008 these changes look like they shouldn't be in here 🤔

Reply:
Hey @MildTomato 
I also felt the same, the only useful section of the code which helped me to solve the issue was 
` html {
  scroll-behavior: smooth;
  width: fit-content;
}`

when i run npm build many of dev-dependencies were not there so they get added to package.json
however i can remove other changes .

and just do the significant code change ` html {
  scroll-behavior: smooth;
  width: fit-content;
}`

Reply:
did you run `npm i` in `/apps/www` or in the root `/`

We use turbo repo now, so npm installing should be done in the root, that might explain.
I'm not sure how this particular file would change though, these look like manual changes.

Reply:
left request here https://github.com/supabase/supabase/pull/9903#discussion_r1007546363 😄